p/scripts/main.js

```diff
@@ -1259,7 +1234,54 @@ function init_crypto_form() {
 }
 //</crypto form (Web login)>
 
+var $sidebar = null;
+var useNiceScrollbar = true;	//TODO: Change to test native vs. JavaScript scrollbars
+
+function sticky_recalc() {
+	if (!$sidebar) {
+		return;
+	}
+	
+	if (!useNiceScrollbar) {
+		return;
+	}
+	var h = 0;
+	if ($nav_entries && $nav_entries.length > 0){
+		h = $(window).height() - $sidebar[0].getBoundingClientRect().top - $nav_entries.height();
+	} else {
+		h = $(window).height() - $sidebar[0].getBoundingClientRect().top;
+	}
+	if (h > 0) {
+		$sidebar.height(h);
+	}
+}
+
+function init_simple_scrollbar() {
+	if (!window.SimpleScrollbar) {
+		if (window.console) {
+			console.log('FreshRSS waiting for simple-scrollbar…');
+		}
+		window.setTimeout(init_simple_scrollbar, 100);
+	} else {
+		SimpleScrollbar.initEl($sidebar[0]);
+	}
+}
 
+var scrollTimeout;
+function init_sticky_sidebar(){
+	$sidebar = $('#sidebar');
+	if (useNiceScrollbar) {
+		init_simple_scrollbar();
+	}
+	$(window).scroll(function () {
+				if (scrollTimeout) {
+					clearTimeout(scrollTimeout);
```

Comment:
Just at a quick glance, it looks like if the user scrolls continuously, the recalculation will never be called (until the scrolling ends), will it? :-)